### DISCUSSION TITLE
Contribution etiquette

### DISCUSSION THREAD

**Discussion by enzocomics** [Automattic/wordpress-activitypub#193] [General]
Hey @pfefferle! Do you have any workflow preferences for how outside developers contribute to the project? I'd love to help where I can, but I've only ever worked on personal repositories in the past, and I'm not familiar with Github etiquette regarding group projects.

Like, if I can see one or several issues I can fix, should I start a discussion thread first? What if the fix is very small, or if it's a a series of small, unrelated changes?

Thanks for your time.

**Comment by pfefferle** [comment]
I have not yet thought about that... the project got a lot of attention in the last few days, so I had not really much time to think about it :)

It would be nice if you would give me a ping, if you want to work on something, so that we do not do things twice. Otherwise, feel free to help me/us to grow the project and bring in some of your ideas.

It seems to be an interesting time for this little beta plugin ;)

**Reply by enzocomics** [reply]
It's very interesting! I'm quite honestly super excited about it. I love the idea of my website's posts and comment section being directly connected to the Fediverse, so things like the discussion threads and the like/favourite counter are mirrored from (for example) Mastodon. 

I'm mainly a  content creator, but sometimes I feel like I spend way too much time scheduling posts and bouncing around various social media platforms trying to keep up with engaging people. Being able to just post once on my WordPress site, and have readers engage with me there without having to leave their preferred platform/client, is basically the dream.

But anyway, thanks for letting me know! I will do my best to keep you in the loop before I attempt to make contributions. I'll probably post them in the discussions/ideas section.

**Reply by KevinGimbel** [reply]
I'd like to contribute as well! My PHP and WordPress is a bit rusty, but I may be able to help with testing stuff or keep an eye on the issue tracker, seeing if I can help people with generic questions, etc. 

This plugin is exactly what I've been looking for and bridges the gap between Fediverse and "own website" almost perfectly already!
